content/geoip/updating-databases.mdx

```diff
@@ -1,6 +1,148 @@
+recommended**).
+
+```bash
+# GeoIP.conf file - used by geoipupdate program to update databases
+# from http://www.maxmind.com
+AccountID YOUR_ACCOUNT_ID_HERE
+LicenseKey YOUR_LICENSE_KEY_HERE
+EditionIDs YOUR_EDITION_IDS_HERE
+```
+
+Note that for `geoipupdate` versions less than 2.5.0, use `UserId` instead of
+`AccountID` and `ProductIds` instead of `EditionIDs`.
+
+### 3. Run GeoIP Update
+
+Run `geoipupdate`. To fully automate this process on Linux or Unix, use a
+crontab file like:
+
+```bash
+# top of crontab
+MAILTO=your@email.com
+
+7 9 * * 3 /usr/local/bin/geoipupdate
+# end of crontab
+```
+
+This crontab file would run every week, and it would email you the results.
+
+If you are running a firewall, `geoipupdate` requires that the DNS and HTTPS
+(443) ports be open.
+
+### Troubleshooting
+
+If you receive an error when running GeoIP Update, you can use the verbose
+option by adding the `-v` flag. This will print out each step taken as it runs.
+The information provided can often help to clarify issues. Additional
+instructions for GeoIP Update can be viewed when running `man geoipupdate` on
+the command line.
+
+Please note that GeoIP Update requires current access to database updates to
+run properly. If your subscription needs to be renewed, updates can be
+purchased while logged into [your Account Summary
+page](https://www.maxmind.com/en/accounts/current/people/current).
+
+Updating to the [latest release of GeoIP
+Update](https://github.com/maxmind/geoipupdate/releases) may be required to
+resolve some errors.
+
+## Directly downloading databases
+
+This is only recommended for systems unable to use GeoIP Update program or for
+clients using the CSV-format databases.
+
+In order to download the databases from a script or program, please use the
+permalinks found on the [GeoIP download
+page](https://www.maxmind.com/en/accounts/current/geoip/downloads).
+
+1. In the “Download Links” column, click “Get Permalink(s)” for the desired
+database.
+2. Copy the permalink(s) provided in the modal window.
+3. Replace `YOUR_LICENSE_KEY` with your license key.
+4. If you are using `wget` or `curl` from a shell script, please be sure to
+quote the URL.
+
+This download will return a compressed file in gzip (for binary) or zip (for
+CSV) format, which can be uncompressed using a program like
+[7-Zip](https://www.7-zip.org/) or [gzip](https://www.gzip.org/).
+
+### Checking for the Latest Release Date
+
+You can check the date of a given database’s latest release by issuing a HEAD
+request for that database’s download permalink URL, as found on our [database
+download page](https://www.maxmind.com/en/accounts/current/geoip/downloads)
+[login required].
+
+For example, using the permalink for the GeoIP2 City CSV database, you can
+issue a command like the following:
+
+```cli
+curl -I 'https://download.maxmind.com/app/geoip_download?edition_id=GeoIP2-City-CSV&amp;license_key=YOUR_LICENSE_KEY&amp;suffix=zip'
```

Comment:
On the current page https://dev.maxmind.com/geoip/geoip-direct-downloads/ the `&` appears by itself without the `amp` text next to it.